### DISCUSSION TITLE
How I can prepare value before render

### DISCUSSION THREAD

**Discussion by xkomiks** [TanStack/table#4923] [Q&A]
For the `boolean` value, I would like to replace as `Yes/No`, this is a primitive example but I have several data structures that are repeated

**Comment by KevinVandy** [comment]
Use the custom `cell` render

**Reply by xkomiks** [reply]
@KevinVandy like this? https://tanstack.com/table/v8/docs/guide/column-defs#cell-formatting

**Reply by xkomiks** [reply]
Can something be done at  `flexRender` level?
